Project-1/README.md

```diff
@@ -10,113 +10,324 @@ Tip #4: The work we do is challenging; that should be assumed. You are smart eno
+(e) An individual has exactly two brothers if and only if they have two siblings who identify as male.
 
 4. Let V be a vocabulary of the attribute (concept) language with complements (ALC) consisting of a role name "parent_of" and a concept name "Male". Interpret parent_of as "x is a parent of y" and M as "x is male". Where possible define the following formulas in this vocabulary; where not possible, explain why: 
   ```(a)  B that says that x is a brother of y
   (b)  A that says that x is an aunt of y
   (c)  C that says that x and y are cousins 
   (d)  O that says that x is an only child
   (e)  T that says that x has exactly two brothers 
-```
 
-5. Select two formulas defined in ALC from question 4 to form the basis of a T-Box. Supplement this T-box with whatever other axioms you like, as well as an A-box, so that you ultimately construct a knowledge base K = (T,A). Provide a _model_ of K. This may be graphical or symbolic or both. 
+  ⊔ ⊓ ⊧ ⊭ ⊦ ⊬ ⊏ ⊐ ⊑ ⊒ C ¬ ≡ ≠ ≥ ≤
+  parent_of
+  m
+
+  (a)  B that says that x is a brother of y
+  (a) $B ≡ M\sqcap ∃parentOf^-.(∃parentOf\ge2)$
+  (a)  B ≡ M ⊓ ∃parentOf^-. (∃parentOf≥2)
+  (a) assuming gender binary determined by chromosomes 
+
+  (b)  A that says that x is an aunt of y
+  (b)  $A ≡ ¬M\sqcap ∃parentOf^-.((∃parentOf.(∃parentOf.M\sqcup ¬M))\sqcap∃parentOf\ge2)$
+  (b)  A ≡ M ⊓ ∃parentOf^-. (∃parentOf.(≥22parentOf.(parentOfM ⊔ ¬M) ⊓ ∃parentOf≥2)
+  (b) assuming gender binary determined by chromosomes 
+
+  (c)  C that says that x and y are cousins 
+  (c)  $C ≡ ∃parentOf^-.(∃parentOf^-.(\ge2parentOf.(parentOf.M\sqcup ¬M)))$
+  (c)  C ≡ ∃parentOf^-.(∃parentOf^-.(≥22parentOf.(parentOf M ⊔ ¬M))
+
+  (d)  O that says that x is an only child
+  (d) $O ≡ ∃parentOf^-.(∃parentOf\le1)$
+  (d) O ≡ ∃parentOf^-.(∃parentOf.≤1)
+
+  (e)  T that says that x has exactly two brothers 
+  (e) $T ≡ (M\sqcap ∃parentOf^-.(\le3∃parentOf.M \sqcap \ge3∃parentOf.M))\sqcup(¬M\sqcap∃parentOf^-.(\le2∃parentOf.M \sqcap \ge2∃parentOf.M))$
+  (e)  T ≡ M ⊓ ∃parentOf^-.(≤3∃parentOf. M) ⊔ (¬M ⊓ ∃parentOf^-.(≤2∃parentOf^-.M ⊓ ≥ 2∃parentOf^-.M))
+
+5. Select two formulas defined in ALC from question 4 to form the basis of a T-Box. Supplement this T-box with whatever other axioms you like, as well as an A-box, so that you ultimately construct a knowledge base K = (T,A). Provide a _model_ of K. This may be graphical or symbolic or both.
+
+ TBox = {Brother ⊆ Male,		(TBox.1)
+	Aunt ⊆ ~Male}		(TBox.2)
+
+Donnie = {Frank: Brother ∩ Male		(Axiom.1)
+	Remi: Brother ∩ Male		(Axiom.2)
+	Mary: Aunt			(Axiom.3)
+	(Donnie, Frank): parent_of	(Axiom.4)
+	(Donnie, Remi): parent_of}		(Axiom.5)
+	
+ΔI = {w, x, y, z},
+William  = w
+Xaivar = x
+Eunice = y
+Zane = z
+Male = {w, x}
+Brother = {w, x}
+Aunt = {y}
+Parent_of = {(z,w), (z,x)}
```

Comment:
It seems you do not specify the value of "Remi", "Mary", "Frank" and "Donnie". Insetad, you assign a value to "William", "Xaivar", "Eunice", and "Zane". Maybe you can somehow change the latter to the former?